syft/frameworks/torch/hook/hook.py

```diff
@@ -492,6 +493,150 @@ def _hook_crypten(self):
         new_func = get_hooked_crypten_func("load", crypten_load)
         setattr(crypten, "load", new_func)
 
+    def _hook_crypten_module(self):
```

Comment:
Q: Maybe it is worth to have an issue created to move the ```CrypTen``` hooking functionality to a different file? (this file would then import specific functions and call them) -- Maybe in that way we have a clear separation between the ```TorchHook``` and the ```CryptenHook```?

Reply:
yeah I definitely think so, and also giving the option to disable crypten hooking (`syft.TorchHook(th, hook_crypten=False)`)

Reply:
Agreed!

Reply:
Created this issue -- maybe we can add it as a comment?
https://github.com/OpenMined/PySyft/issues/3385